dynomitemanager/src/main/java/com/netflix/dynomitemanager/sidecore/utils/RetryableCallable.java

```diff
@@ -37,6 +37,8 @@ public RetryableCallable()
 
     public RetryableCallable(int retrys, long waitTime)
     {
+    	if(retrys<=-1) throw new RuntimeException(new IllegalAccessException("Retrys should be >= 0"));
```

Comment:
Yes safety check and validation. IF this is retry based IMHO needs to be 0 in other words no retry or any positive number.